mlfinlab/bet_sizing/.ipynb_checkpoints/bet_sizing_tutorial-checkpoint.ipynb

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
Please don't add these checkpoints. As far as I know, they can be removed without breaking the structure of the notebook?

Reply:
ipynb checkpoint removed.